packages/amazon-sumerian-hosts-babylon/src/Babylon.js/HostObject.js

```diff
@@ -84,6 +87,561 @@ class HostObject extends CoreHostObject {
 
     event.notifyObservers(value);
   }
+
+  /**
+   * @private
+   */
+  static async initTextToSpeech() {
```

Comment:
nit: This looks like a function which could sit in core HostObject class. Although, i guess in that case we should evaluate the necessity of letting user pass in their own aws service object or we always create for them like we do here.